packages/react-native-session-replay/package.json

```diff
@@ -16,6 +16,7 @@
     },
     "license": "Apache-2.0",
     "main": "lib/commonjs/index",
+    "private": "true",
```

Comment:
is it a valid change?

Reply:
It was removed by mistake with the previous release. We need this to prevent publishing a live version by mistake, as Session Replay is released separately as alpha.